src/Components/ServiceScene/Breakdowns/AddToFiltersButton.tsx

```diff
@@ -43,12 +44,17 @@ export function addToFilters(
   value: string,
   operator: FilterType,
   scene: SceneObject,
-  variableType?: VariableFilterType
+  variableType?: VariableFilterType,
+  skipFavorites?: boolean
```

Comment:
This is gross, I'm adding implementation to prevent from ripping out more unit tests.

Reply:
Definitely. Let's not do this.

Reply:
You can mock '../../../services/store', doesn't that work?

Reply:
Yeah good call, I was running on fumes last night. Fixed!